src/assets/recipeData.json

```diff
@@ -6749,51 +6750,51 @@
   "Yazaghor": {
     "material": "Yazaghor",
     "craftable": false,
-    "stat": "Shard",
+    "stat": "shard",
     "faction": "Thousand Sons",
-    "rarity": "Shard",
+    "rarity": "shard",
     "locations": [
       "SH45"
     ]
   },
   "Toth": {
     "material": "Toth",
     "craftable": false,
-    "stat": "Shard",
+    "stat": "shard",
     "faction": "Thousand Sons",
-    "rarity": "Shard",
+    "rarity": "shard",
     "locations": [
       "SH15"
     ]
   },
   "Abraxas": {
     "material": "Abraxas",
     "craftable": false,
-    "stat": "Shard",
+    "stat": "shard",
     "faction": "Thousand Sons",
-    "rarity": "Shard",
+    "rarity": "shard",
     "locations": [
       "SH30"
     ]
   },
   "Thaumachus": {
     "material": "Thaumachus",
     "craftable": false,
-    "stat": "Shard",
+    "stat": "shard",
     "faction": "Thousand Sons",
-    "rarity": "Shard",
+    "rarity": "shard",

```

Comment:
I think for "shards" we can put initial char rarity e.g. Celestine - Legendary, Yazaghor - Common

Reply:
That's not a problem.

I did as shard so that way so that 

campaignCofig(battleData(recipeData("item").location[x]).campaignType).dropRate(recipeData("item").rarity) would give the right drop rate regardless of if it was a material or a shard.

Reply:
yeap you right and I already used it like that, then leave it as it is for now